2 commits, an initial translation of c00/30/04->14 and an extra adding sounds as per indication. planning to use dupe tool after this (#63)
IllusionMods/HS2-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/12/2/translation.txt

//
// Dumped for Honey Select 2 v1.1.3 by Text Dump v1.4.1
//

> 大変おいしゅうございました。ああ、ご挨拶が遅れて申し訳ございません。おはようございます、今日も素敵な一日になると良いですね。=It was extremely delicious. Oh, I'm sorry for the late greeting. Good morning, I hope you have a wonderful day.

Reviewer: ```suggestion
大変おいしゅうございました。ああ、ご挨拶が遅れて申し訳ございません。おはようございます、今日も素敵な一日になると良いですね。=That was very tasty. Oh, sorry for not greeting you sooner. Good morning, I hope you have a wonderful day.
```

> おはようございます。……普通に起こすつもりだったのですが、膨らみに気づいた瞬間、己の欲望を抑えきれませんでした。=Good morning... I was going to wake you up normally, but when I saw the bulge, I couldn't control my desire.

Reviewer: ```suggestion
おはようございます。……普通に起こすつもりだったのですが、膨らみに気づいた瞬間、己の欲望を抑えきれませんでした。=Good morning... I was going to wake you up normally, but when I saw the bulge, I couldn't hold back.
```
